notifyre.sh

```diff
@@ -46,23 +40,21 @@ notif_timer() {
   if [ "$elapsed" -gt "$MIN_INTERVAL" ]; then
     if [ "$(uname -s)" = "Darwin" ]; then
       # Use terminal-notifier for Mac OS
-      terminal-notifier -title "NotiFyre $(echo $command_status)"\
-        -subtitle "Command : $commandx"\
-        -message "Completed in $elapsed seconds"\
-        -timeout 5\
+      terminal-notifier -title "NotiFyre $(echo $command_status)" \
+        -subtitle "Command : $commandx" \
+        -message "Completed in $elapsed seconds" \
+        -timeout 5 \
         -closeLabel "Gotcha!"
     else
       # Use notify-send for others
-      notify-send "NotiFyre $(echo $command_status)" "Terminal in ${p[-2]}/${p[-1]} and exit $? \$" \
-        "completed $commandx in $elapsed seconds" \
-        -i utilities-terminal
-        -t 50
+      notify-send "NotiFyre $(echo -e $command_status)" \
+        "Completed $commandx in $elapsed seconds" \
+        -i utilities-terminal \
+        -t 5000
     fi
 
-    if [ "$SOUND" -eq 1 ]; then
-      if [ "$elapsed" -gt "$SOUND_MIN" ]; then
-        paplay $ALERT
-      fi
+    if [ "$SOUND" -eq 1 ] && [ "$elapsed" -gt "$SOUND_MIN" ]; then
+        paplay $ALERT &> /dev/null || echo -en "\a";
```

Comment:
Want to confirm if `paplay` works properly, as well as its fallback

Reply:
`paplay` should probably be used only while in non-Mac conditions.

Reply:
That's fine in that case.
Could you or anyone test if this works ? With a valid alert tone provided to `paplay`

Reply:
@a7r3 It works in bash,

@athityakumar Did not understand your point

Reply:
Thanks! @kaustubhhiware

Reply:
@kaustubhhiware Probably he meant that `paplay` does not exist in a Mac System

Reply:
@a7r3 - That's what I meant. 😄

Reply:
@athityakumar Weird, you never mentioned earlier.

Why don't you add an OSX alternative then, like before? (that OS check =="DARWIN") [If that is possible]

Reply:
@kaustubhhiware @athityakumar 

For now, the fallback i.e. `echo -e "\a"` should work on Mac Systems (Googled it)

Till then, you may find a workaround for playing custom notification files